internal/plan/plan.go

```diff
@@ -149,6 +151,47 @@ func (a *Action) DetailedString() string {
 	}
 }
 
+// Apply commits all required changes for action to given Host.
+func (a *Action) Apply(ctx context.Context, host hostPkg.Host) error {
+	args := []any{}
+	diffStr := a.DiffString()
+	if len(diffStr) > 0 {
+		args = append(args, []any{"diff", diffStr}...)
```

Comment:
Why `any` can’t the diff be all strings ?

Reply:
Because [slog `Info` interface](https://pkg.go.dev/log/slog#Logger.Info) expects pairs of key / values for [attrs](https://pkg.go.dev/log/slog#Attr). Check the example [here](https://pkg.go.dev/log/slog#hdr-Attrs_and_Values).